WORKSPACE

```diff
@@ -0,0 +1,13 @@
+load("//bazel:repositories.bzl", "envoy_dependencies")
+
```

Comment:
Turns out that if you move bazel to tools/bazel, Bazel tries to invoke the directory as a binary. I'm guessing that similar to the special nature of tools/bazel.rc, Bazel looks in tools/bazel for another bazel binary to reinvoke?!